python/ray/autoscaler/node_provider.py

```diff
@@ -208,3 +210,9 @@ def terminate_nodes(self, node_ids):
     def cleanup(self):
         """Clean-up when a Provider is no longer required."""
         pass
+
+    def get_command_runner(self, log_prefix, node_id, provider, auth_config,
+                           cluster_name, process_runner, use_internal_ip):
+        """ Returns the CommandRunner class used to perform SSH commands. """
```

Comment:
Can you document the arguments in pydoc format:
"""

Args:
  log_prefix (str): What this is...
  ...
"""
?

Reply:
Shouldn't this be documented in the class definitions?